src/filesystem/impls/appshell/node/FileWatcherDomain.js

```diff
@@ -99,7 +99,8 @@ function watchPath(path, ignored) {
             ignoreInitial: true,
             ignorePermissionErrors: true,
             followSymlinks: true,
-            ignored: ignored
+            ignored: ignored,
+            usePolling: process.platform === "win32"
```

Comment:
sadly, `chokidar` causes watched files to lock-up, can't use `npm install` or `bower install` over a watched directory, will need to investigate more